lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -776,72 +773,85 @@ func (c *ContainerProxy) AttachStreams(ctx context.Context, vc *viccontainer.Vic
 		}
 	}
 
+	var inCtx context.Context
+	var inCancel, cancel func()
 	if ca.UseStdin {
 		wg.Add(1)
+		inCtx, inCancel = context.WithCancel(ctx)
+		defer inCancel()
```

Comment:
I can't remember but does the AttachServer keep stdin open when we detach?  There is a parameter named OpenStdin in the config.  I'm not sure we are handling it.  We did at one time honor it in the personality server but a few people have touched this code and I don't really know the current state.